cookbook/configuration/mongodb_session_storage.rst

```diff
@@ -0,0 +1,54 @@
+How to Use MongoDbSessionHandler to Store Sessions in a MongoDB Database
+========================================================================
+
+The default Symfony session storage writes the session information to files.
+Some medium to large websites use a NoSQL database called MongoDB to store the
+session values instead of files, because databases are easier to use and scale
+in a multi-webserver environment.
+
+Symfony has a built-in solution for NoSQL database session storage called
+:class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MongoDbSessionHandler`.
+MongoDB is an open-source document database that provides high performance,
+high availability, and automatic scaling. This article assumes that you have
```

Comment:
Minor correction: we don't use the so-called "serial comma", so the comma before the `and` word must be removed. Thanks.